src/tir/transforms/lower_sparse_tir.cc

```diff
@@ -67,19 +99,44 @@ class AccessAndDependencyCollector : public StmtExprVisitor {
       for (int k = 0; k < ndim; ++k) {
         const SpIterVar& sp_iter = kv_pair.second[k];
         if (sp_iter->kind == SpIterKind::kDenseFixed ||
-            sp_iter->kind == SpIterKind::kDenseVariable ||
```

Comment:
Does that mean we need to collect the iterators DenseVariable iterators depend on?

Reply:
Currently, yes. But we seem to agree that the analysis needs improvement.